doc/content/reference/api/end_device.md

```diff
@@ -4,59 +4,114 @@ description: ""
 weight: 7
 ---
 
+End devices are registered in multiple registries. The Identity Server has a registry with end device metadata, the Network Server's registry contains the MAC configuration and MAC state (including network session keys), the Application Server keeps payload formatters and application session keys, the Join Server keeps the root keys.
```

Comment:
Network session keys are not part of MAC state, so I think it's better to do the same as with AS:
```suggestion
End devices are registered in multiple registries. The Identity Server has a registry with end device metadata, the Network Server's registry contains the MAC configuration, MAC state, and network session keys, the Application Server keeps payload formatters and application session keys, the Join Server keeps the root keys.
```